docs/manifest.json

```diff
@@ -161,353 +71,443 @@
+	},
+	{
+		"title": "Developer Contributions",
+		"slug": "develop",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/develop.md",
+		"parent": "contributors"
+	},
+	{
+		"title": "Coding Guidelines",
+		"slug": "coding-guidelines",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/coding-guidelines.md",
+		"parent": "develop"
+	},
+	{
+		"title": "Testing Overview",
+		"slug": "testing-overview",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/testing-overview.md",
+		"parent": "develop"
+	},
+	{
+		"title": "Git Workflow",
+		"slug": "git-workflow",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/git-workflow.md",
+		"parent": "develop"
+	},
+	{
+		"title": "Block Grammar",
+		"slug": "grammar",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/grammar.md",
+		"parent": "develop"
+	},
+	{
+		"title": "Scripts",
+		"slug": "scripts",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/scripts.md",
+		"parent": "develop"
+	},
+	{
+		"title": "Gutenberg Release Process",
+		"slug": "release",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/release.md",
+		"parent": "develop"
+	},
+	{
+		"title": "Documentation Contributions",
+		"slug": "document",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/document.md",
+		"parent": "contributors"
+	},
+	{
+		"title": "Copy Guidelines",
+		"slug": "copy-guide",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/copy-guide.md",
+		"parent": "document"
+	},
+	{
+		"title": "History",
+		"slug": "history",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/history.md",
+		"parent": "contributors"
+	},
+	{
+		"title": "Glossary",
+		"slug": "glossary",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/glossary.md",
+		"parent": "contributors"
+	},
+	{
+		"title": "Frequently Asked Questions",
+		"slug": "faq",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/faq.md",
+		"parent": "contributors"
+	},
+	{
+		"title": "Repository Management",
+		"slug": "repository-management",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/repository-management.md",
+		"parent": "contributors"
+	},
+	{
+		"title": "Outreach",
+		"slug": "outreach",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/outreach.md",
+		"parent": "contributors"
+	},
 	{
 		"title": "Tutorials",
```

Comment:
This is what @dd32 taught me about redirects:

- The code for the redirects lives [here](https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/themes/pub/gutenberg/functions.php?marks=856#L850).
- It is possible to set up manual redirects (see code above).
- If the slug of the page (last bit in `/this/is/the/SLUG/`) remains the same it'll be redirected automatically.

The changes introduced in this PR shorten the page URL removing the `designers-developers/developers` part, so my understanding is that they'll be redirected automatically:

Section | Before | After
--- | --- | ---
Tutorials (+33 subsections) | https://wordpress.org/gutenberg/handbook/designers-developers/developers/tutorials/ | https://wordpress.org/gutenberg/handbook/tutorials/
Block API (+6 subsections) | https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/ | https://wordpress.org/gutenberg/handbook/block-api/
Components (+65 subsections) | https://wordpress.org/gutenberg/handbook/designers-developers/developers/components/ | https://wordpress.org/gutenberg/handbook/components/
Data (+9 subsections) | https://wordpress.org/gutenberg/handbook/designers-developers/developers/data/ | https://wordpress.org/gutenberg/handbook/data/
Filters (+4 subsections) | https://wordpress.org/gutenberg/handbook/designers-developers/developers/filters/ | https://wordpress.org/gutenberg/handbook/filters/
Packages (+19 subsections) | https://wordpress.org/gutenberg/handbook/designers-developers/developers/packages/ | https://wordpress.org/gutenberg/handbook/packages/

Reply:
Got confirmation from Dion that this holds true. If any particular section fails to redirect we can fix it in a case by case basis. We don't expect any to fail, though, and I've tested that the proposed changes are already redirected.